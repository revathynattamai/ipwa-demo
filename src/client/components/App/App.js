import React from "react";
import { StyleSheet, View } from "react-native";

function App() {
  return (
    <View style={styles.app}>
    </View>
  );
}

const styles = StyleSheet.create({
  app: {
    height: '100vh',
    backgroundColor: '#000',
  },
});

export default App;
