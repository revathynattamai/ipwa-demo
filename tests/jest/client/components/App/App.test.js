import React from 'react';
import App from '../../../../../src/client/components/App/App';
import renderer from 'react-test-renderer';

test('App -> render', () => {
  const component = renderer.create(
    <App />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
