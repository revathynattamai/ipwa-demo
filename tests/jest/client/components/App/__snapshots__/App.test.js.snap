// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App -> render 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#000",
      "height": "100vh",
    }
  }
/>
`;
